src/course-outline/highlights-modal/HighlightsModal.jsx

```diff
@@ -23,7 +23,7 @@ const HighlightsModal = ({
   onSubmit,
 }) => {
   const intl = useIntl();
-  const { highlights = [], displayName } = useSelector(getCurrentSection);
+  const { highlights = [], displayName } = useSelector(getCurrentItem);
   const initialFormValues = getHighlightsFormValues(highlights);
 
```

Comment:
Nit: please correct me if I am wrong, but highlights are specific to sections. Any reason to switch to `getCurrentItem` ?

Reply:
No, actually `getCurrentSection` is better. Its just that I added `getCurrentSection` at a later point for delete option.